cms/templatetags/cms_wizard_tags.py

```diff
@@ -11,6 +12,7 @@
 register = template.Library()
 
 
+@register.tag(name='cms_wizard')
```

Comment:
Removing it in a patch release of an LTS looks *risky* to me, but I'm -0 on removing it